src/cache/httpLink.js

```diff
@@ -0,0 +1,7 @@
+import { HttpLink } from 'apollo-link-http';
+
+const httpLink = new HttpLink({
+  uri: 'http://potato-store.herokuapp.com/graphql',
```

Comment:
It should be possible to specify another backend through environment variables, as earlier implemented in general.js